.github/workflows/test_dev_env.yml

```diff
@@ -125,4 +125,5 @@ jobs:
       - name: Zeitwerk
         run: bin/rails zeitwerk:check
       - name: yarn npm audit
-        run: yarn npm audit
+        run: yarn npm audit --ignore 1098356
```

Comment:
this is a complaint about a bootstrap thing with no remediation, so ignoring for now